homeassistant/components/qvr_pro/camera.py

```diff
@@ -90,6 +90,13 @@ def device_state_attributes(self):
 
     def camera_image(self):
         """Get image bytes from camera."""
+        try:
+            return self._client.get_snapshot(self.guid)
+
+        except QVRResponseError as ex:
```

Comment:
Is this always a connection issue?

Reply:
No.  We only want to reconnect when there is a failure, so we catch the exception, reinitialize the connection and try the call again after reconnecting.

Reply:
@balloob If there is some better way to handle this, please let me know.  Python is just a hobby language for me.